### PR TITLE
Add AWSElasticBeanstalkMulticontainerDocker role for ec2 instance profile

### PR DESCRIPTION
**Pull request by aj-jaswanth-ee**
Fixes https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment/issues/79

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -82,6 +82,15 @@ data "aws_iam_policy_document" "ec2" {
   }
 }
 
+data "aws_iam_policy" "AWSElasticBeanstalkMulticontainerDocker" {
+  arn = "arn:aws:iam::aws:policy/AWSElasticBeanstalkMulticontainerDocker"
+}
+
+resource "aws_iam_role_policy_attachment" "AWSElasticBeanstalkMulticontainerDocker" {
```
Comment (aknysh) [inline main.tf]:
```suggestion
resource "aws_iam_role_policy_attachment" "elastic_beanstalk_multi-container-docker" {
```

Comment (aknysh) [reply main.tf]: use snake-lower-case to name resources

File: main.tf [hunk 2]
```diff
@@ -82,6 +82,15 @@ data "aws_iam_policy_document" "ec2" {
   }
 }
 
+data "aws_iam_policy" "AWSElasticBeanstalkMulticontainerDocker" {
```
Comment (aknysh) [inline main.tf]: why do we need to lookup if we already know the ARN?

**Comment by aj-jaswanth-ee** [comment]
@aknysh Sorry, that was me in auto pilot mode. Applied all suggestions.
Thanks for maintaining all the modules.

**Pull request merged**